errai-ioc/src/main/java/org/jboss/errai/ioc/client/api/EnabledByProperty.java

```diff
@@ -43,4 +43,20 @@
    * if the specified system property exists and has the value {@code "true"}.
    */
   boolean negated() default false;
+
+  /**
+   * When enabled, the current value is used as default value in case the system property is not set.
```

Comment:
IMO, it's unclear what "current value" means here. I think it would be more clear as:

    When enabled, the matchValue is used as the default value in case the system property is not set.